Apps/Src/ReadCarCAN.c

```diff
@@ -5,45 +5,69 @@
 #include "Contactors.h"
 #include "Minions.h"
 #include "CAN_Queue.h"
+#include "FaultState.h"
+#include "os_cfg_app.h"
 
-static bool msg_recieved = false;
-static OS_MUTEX msg_rcv_mutex;
+// Saturation threshold is halfway between 0 and max saturation value (half of summation from one to the number of positions)
+#define saturationThreshold (SAT_BUF_LENGTH + 1) * SAT_BUF_LENGTH / 4 
```

Comment:
It would actually evaluate correctly as is, since `saturationThreshold` is never used in any mathematical expression. But it's certainly good practice to put it in parentheses.